### DISCUSSION TITLE
Kanban Style View with toggle

### DISCUSSION THREAD

**Discussion by Greenlight88** [cortexrd/Knack-Toolkit-Library#385] [Ideas]
Would be great to display some views built for mobile as a kanban style and toggle through items in a list, or even better -  use the swipe feature on mobiles? Many use cases but say for others - swipe through a list of available products with picture and 'Order Now' Button - we could use the existing Ktl mobile class to only display this view on Mobile

**Comment by cortexrd** [comment]
I was planning to add new capabilities to the **_dnd** keyword (Drag n Drop), so it would be possible to drag an element from one view to another.  Would that do what your describing?

Example:

**view_1** would be a list of items for sale
**view_2** would start empty being your shopping cart.

You'd drag items from view_1 to view_2 until you have all you need, then click the Purchase Now button to check out.

**Comment by Greenlight88** [comment]
Not exactly.. though that sounds great 

I think Knack can almost do what I have in mind with a List View - its more about the presentation of that list and how it presents in particular on a mobile - basically just a card style presentation of an item from a grid of items - but in the mobile view you can hit an arrow to get to next page (or more ideally swipe across) - allowing for a nice presentation of items in a list - each one might have a photo, some details and a link - you could definitely use this in a shopping situation with a button to add to cart or something


here is an example from the web

![Screenshot 2024-10-16 101948](https://github.com/user-attachments/assets/3d3de0bb-3d26-44de-9bad-01e633d2e883)

**Reply by Greenlight88** [reply]
But imagine there is only one item instead and you had little arrows or something either side to toggle between items

**Comment by Greenlight88** [comment]
I think the word im looking for here is 'carousel'


https://github.com/user-attachments/assets/2029dace-8a1b-4679-bc9b-0384689b462e

This is a better example
